pages/web/domains/dns_zonemaster/guide.en-gb.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Using Zonemaster'
+slug: ovhcloud-domain-zonemaster-tutorial
+section: DNS and DNS zone
+order: 08
+---
+
+**Last updated 12th September 2022**
+
+> [!warning]
+>
+> OVHcloud provides services which you are responsible for with regard to their configuration and management. You are therefore responsible for ensuring they function correctly.
+> 
+> We have provided you with this guide in order to help you with common tasks. Nevertheless, we recommend contacting a specialist [provider](https://partner.ovhcloud.com/en-gb/) if you encounter any difficulties. We will not be able to assist you ourselves. You can find more information in the [Go further”](#go-further) section of this guide.
+> 
+
+## Objective
+
+[Zonemaster](https://zonemaster.net/domain_check) is a tool created by the collaboration between [AFNIC](https://www.afnic.fr/en/) (French registry) and [The Swedish Internet Foundation](https://internetstiftelsen.se/en/) (Swedish registry). It allows you to analyse the Domain Name System (DNS) configuration of a domain name and identify what can be improved or corrected.
+
+> [!primary]
+>
+> To get a better understanding of DNS, please read the introduction to our guide on [configuring a DNS](https://docs.ovh.com/gb/en/domains/web_hosting_how_to_edit_my_dns_zone/) zone.
+
+## Requirements
+
+- a [domain name](https://www.ovhcloud.com/en-gb/domains/)
+
+## Instructions
+
+### Input field
+
+The Zonemaster tool allows you to check a DNS configuration in place on a domain name or to test a preconfigured DNS zone on future DNS servers.
+
+To check the current configuration of a domain name, enter your domain name, then click `check`{.action}
+
+![domains](images/zonemaster01.png){.thumbnail}
+
+To check a DNS configuration that has been prepared but not yet applied to the domain name concerned, select the `Options`{.action} box, and then enter the following information:
+
+- **DNS** servers: Enter the information for the DNS server associated with a domain name, then click the `+`{.action} button to confirm your entry. Entering an IP address is optional.
```

Comment:
please check options names on EN version of Zonemaster

Reply:
same as guide